carousel/docs/getting_started.rst

```diff
@@ -124,11 +124,21 @@ Therefore models can be created three different ways.
         """
         Layers specified as class attributes
         """
-        data = [(MyModelData, {'filename': 'data.json'}), ...]
-        outputs = [MyModelOutputs, ...]
-        formulas = [MyModelFormulas, ...]
-        calculations = [MyModelCalculations, ...]
-        simulations = [MyModelSimulations]
+        data = ModelParameter(
+            layer='Data', sources=[(MyModelData, {'filename': 'data.json'}), ...]
+        )
+        outputs = ModelParameter(
+            layer='Outputs', sources=[MyModelOutputs, ...]
+        )
+        formulas = ModelParameter(
+            layer='Formulas', [MyModelFormulas, ...]
```

Comment:
So change these to all be keyword, like the `data` and `outputs`, which is more explicit and [the Zen of Python says that "explicit is better than implicit"](https://www.python.org/dev/peps/pep-0020/#id3)

```python
    formulas = ModelParameter(
        layer='Formulas', sources=[MyModelFormulas, ...]
    )
    calculations = ModelParameter(
        layer='Calculations', sources=[MyModelCalculations, ...]
    )
    simulations = ModelParameter(
        layer='Simulations', sources=[MyModelSimulations]
    )
```